Sources/SwiftParser/CollectionNodes+Parsable.swift

```diff
@@ -58,7 +58,7 @@ extension AccessorDeclListSyntax: SyntaxParseable {
       return parser.parseAccessorList() ?? RawAccessorDeclListSyntax(elements: [], arena: parser.arena)
     } makeMissing: { remainingTokens, arena in
       return RawAccessorDeclSyntax(
-        attributes: nil,
+        attributes: RawAttributeListSyntax(elements: [], arena: arena),
```

Comment:
Because that would require access to the parser, which would be an overlapping access to `parser`. So to actually use `emptyCollection`, we would need to pass the parser into `makeMissing` and I didn’t want to change that since creation of an empty collection is sufficiently simple here and shouldn’t happen in normal code.